doc/source/computation.rst

```diff
@@ -169,6 +168,7 @@ DataFrame objects.
    df1 = pd.DataFrame(np.random.randn(5, 4), index=index, columns=columns)
    df2 = pd.DataFrame(np.random.randn(4, 4), index=index[:4], columns=columns)
    df1.corrwith(df2)
+   df1.corrwith(df2, how='all')
```

Comment:
can you add a separate sub-section with an example here (and ref the whatsnew to here)

Reply:
Done.